src/lib/slacko.ml

```diff
@@ -248,8 +248,8 @@ type channel_obj = {
   creator: user;
   is_archived: bool;
   is_general: bool;
+  name_normalized: string;
```

Comment:
I guess `conversations.list` returns a `conversation` and not a `channel_obj`, so this needs to be updated somehow.

Reply:
It returns "a list of limited channel-like [conversation objects](https://api.slack.com/types/conversation)".

Reply:
In such case I suggest renaming it to `conversation_obj` since the Slack API calls it the "conversation type".

I think in this case the API breakage is happening anyway so might as well stick to the naming that Slack uses so it is easier to understand.

Reply:
Looks like `channels.*` API was [superseded](https://api.slack.com/docs/conversations-api) along with `im.*` and others by `conversations.*`.  However, there are still `channel`, `group`, `im`, `mpim` [API object types](https://api.slack.com/types), that are different from `conversation` type.  The codebase also has a `conversation` type, which is not the type for the `conversation` API object.  It seems instead of attempting a fix to the notify CLI tool now, we need to overhaul the underlying types first.